sdk/python/kfp/cli/run.py

```diff
@@ -100,4 +100,19 @@ def _display_run(client, namespace, run_id, watch):
 def _print_runs(runs):
     headers = ['run id', 'name', 'status', 'created at']
     data = [[run.id, run.name, run.status, run.created_at.isoformat()] for run in runs]
-    print(tabulate(data, headers=headers, tablefmt='grid'))
\ No newline at end of file
+    print(tabulate(data, headers=headers, tablefmt='grid'))
+
+@run.command()
+@click.argument('run-id')
+@click.pass_context
+def delete(ctx, run_id):
+    """delete a KFP run"""
+    client = ctx.obj['client']
+    # KFP backend API DeleteRun() returns google.protobuf.Empty
```

Comment:
Could you add a delete_run method in the client with the try-catch? I want to minimize the client logic in cli.

Reply:
This is a bug in swagger codegen and we should try to fix it.

Reply:
I've researched the issue and fixed the problem with swagger codegen. See https://github.com/kubeflow/pipelines/issues/1685

Run `pip3 install kfp-server-api==0.1.18.3` to update the package.

There is no need to use try/catch now.

Reply:
It's still better to create a delete_run method in the client module for better consistency and discoverability, even though it's just call the runs.delete_run method. I think tfx team also had some issue by directly calling runs.delete_run which causes linter unhappy.